app/exporters/response_exporter.rb

```diff
@@ -20,8 +22,32 @@ def export_lines(questionnaire_name)
       end
     end
 
+    def export_headers(questionnaire, bust_cache: false)
+      RedisCachedCall.cache("#{QUESTIONNAIRE_HEADERS_KEY}_#{questionnaire.key}", bust_cache) do
+        headers = {}
+        silence_logger do
+          filled_out_questionnaire_responses(questionnaire).find_each do |response|
+            next if Exporters.test_phone_number?(response.protocol_subscription.person.mobile_phone)
+
+            # Response has a .values function, which we are using here (i.e., it is not a hash from which we get the
+            # values)
+            response.values.each do |key, _value|
+              headers[key] = ''
+            end
+          end
+        end
+        sort_and_add_default_header_fields(headers)
+      end
+    end
+
     private
 
+    def filled_out_questionnaire_responses(questionnaire)
```

Comment:
In Questionnaire zou je kunnen maken  

```
has_many :responses, through: :measurement
```

en dan kun je doen `questionnaire.responses.completed`? (of werkt dat niet?

Reply:
Fixed.